modules/ROOT/pages/custom-basic-toolbar-button.adoc

```diff
@@ -17,6 +17,9 @@ include::partial$misc/admon-predefined-icons-only.adoc[]
 |enabled |boolean |optional |default: `true` - Represents the button's state. When `false`, the button is unclickable. Toggled by the button's API.
 |onSetup |`+(api) => (api) => void+` |optional |default: `+() => () => {}+` - Function invoked when the button is rendered. For details, see: xref:using-onsetup[Using `+onSetup+`].
 |onAction |`+(api) => void+` |required |Function invoked when the button is clicked.
+4+|
```

Comment:
That's for the colspan so that it takes up 4 columns.

<img width="1479" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/111734091/ac531bd1-6387-4993-b6ed-0772273b8b90">